Add style for left border highlights on activity feed

jeremiak opened the pull request:
Continuation of #119 to add [left border highlights](https://github.com/18F/cg-dashboard/issues/515#issuecomment-238965105)

Commit:
Add style for left border highlights on activity feed

Commit:
Add style for log activity items

Commit:
Wrap long activity log items

Some of the items generated from the log endpoint have
requested URLs that are pretty long. To make them more
legible in the interface I'm wrapping the long lines

msecret commented on src/css/components/activity_log.scss:
```diff
@@ -7,21 +12,38 @@
   flex-wrap: wrap;
   margin: 0;
-  padding-left: 2em;
+  padding-left: 1em;
```
Why are these `em` instead of `rem`? There's not font size set here, is there one higher up?

Commit:
Change to rem and fix doc classses

Changes ems to rems because there's not reason to use ems here as
things shouldn't scale by font size.

Also fix classes in the docs part as they were based off old class
names.
